events/stream.go

```diff
@@ -100,11 +116,12 @@ func NewDisposableStream(name string, subjects []string, maxMessages int64) Stre
 	}, []nats.SubOpt{
 		nats.AckNone(),
```

Comment:
wasn't it `ExplicitAck`?

Reply:
Had some troubles when testing which returned weird errors. This eventually had to do with the fact a relative testDir was used instead of a tmp testDir.....